frontend/src/app/main/data/workspace/persistence.cljs

```diff
@@ -55,6 +55,13 @@
                           (rx/debounce 2000)
                           (rx/merge stoper forcer))
 
+            local-file? #(let [event-file-id (:file-id %)]
+                           (or (nil? event-file-id)
+                               (= event-file-id file-id)))
+            library-file? #(let [event-file-id (:file-id %)]
+                           (and (some? event-file-id)
+                                (not= event-file-id file-id)))
+
```

Comment:
A more idiomatic way is using `as->` macro:

```clojure
library-file? #(as-> % event-file-id
                 (and (some? event-file-id)
                      (not= event-file-id file-id)))
```
In any case, the resulting code is 99.9% equal, this is just a **cosmetic change**

Reply:
done